engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -95,6 +97,19 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    @Override
+    public void setUri(String nodeUri) {
```

Comment:
Yes, that is what I meant.

At least, that's my understanding of how public interfaces in Java work.